src/getHumanAge.test.js

```diff
@@ -1,10 +1,22 @@
 'use strict';
 
-describe('getHumanAge', () => {
-  const { getHumanAge } = require('./getHumanAge');
+const { getHumanAge } = require('./getHumanAge');
 
+describe('getHumanAge', () => {
   test('should be declared', () => {
-    expect(getHumanAge)
-      .toBeInstanceOf(Function);
+    expect(getHumanAge).toBeInstanceOf(Function);
+  });
+});
+
+describe('convertToHuman', () => {
```

Comment:
The 'describe' block should be combined with the 'getHumanAge' block, as it is testing the same function.